[2025-09-06T05:01:59.932+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_insert_select.validate_model manual__2025-09-06T05:01:40.198848+00:00 [queued]>
[2025-09-06T05:01:59.950+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_insert_select.validate_model manual__2025-09-06T05:01:40.198848+00:00 [queued]>
[2025-09-06T05:01:59.950+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2025-09-06T05:01:59.974+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): validate_model> on 2025-09-06 05:01:40.198848+00:00
[2025-09-06T05:01:59.987+0000] {standard_task_runner.py:57} INFO - Started process 1042 to run task
[2025-09-06T05:01:59.993+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'mysql_insert_select', 'validate_model', 'manual__2025-09-06T05:01:40.198848+00:00', '--job-id', '255', '--raw', '--subdir', 'DAGS_FOLDER/mysql_insert_select.py', '--cfg-path', '/tmp/tmpcxfwxle5']
[2025-09-06T05:01:59.997+0000] {standard_task_runner.py:85} INFO - Job 255: Subtask validate_model
[2025-09-06T05:02:00.078+0000] {task_command.py:410} INFO - Running <TaskInstance: mysql_insert_select.validate_model manual__2025-09-06T05:01:40.198848+00:00 [running]> on host 6faa580ba6cb
[2025-09-06T05:02:00.234+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_insert_select' AIRFLOW_CTX_TASK_ID='validate_model' AIRFLOW_CTX_EXECUTION_DATE='2025-09-06T05:01:40.198848+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-06T05:01:40.198848+00:00'
[2025-09-06T05:02:00.237+0000] {logging_mixin.py:149} INFO - ✅ Modelo cargado correctamente
[2025-09-06T05:02:00.237+0000] {logging_mixin.py:149} INFO -    - Tipo: <class 'numpy.ndarray'>
[2025-09-06T05:02:00.237+0000] {logging_mixin.py:149} INFO -    - Características esperadas: No disponible
[2025-09-06T05:02:00.238+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-09-06T05:02:00.254+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=mysql_insert_select, task_id=validate_model, execution_date=20250906T050140, start_date=20250906T050159, end_date=20250906T050200
[2025-09-06T05:02:00.327+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-09-06T05:02:00.377+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
